src/editing_opr/image_opr.py

```diff
@@ -0,0 +1,25 @@
+import cv2
```

Comment:
Please remove the upstream files that are not being used.

Reply:
https://github.com/georgia-tech-db/Eva/pull/36#discussion_r355106785

Reply:
https://github.com/georgia-tech-db/Eva/pull/36#discussion_r355106982